.travis.yml

```diff
@@ -37,6 +37,6 @@ install:
 
 script:
   - ls -l $TRAVIS_BUILD_DIR
-  - docker run -v $TRAVIS_BUILD_DIR:/ws/incubator-mnemonic --rm -t mnemonic/mneci /bin/bash -c "cd /ws/incubator-mnemonic; git log -1 --stat; git clean -xdf; mvn clean install"
+  - docker run -v $TRAVIS_BUILD_DIR:/ws/incubator-mnemonic --rm -t mnemonic/mneci /bin/bash -c "cd /ws/incubator-mnemonic; git log -1 --stat; git clean -xdf; mvn clean install; mvn javadoc:javadoc"
```

Comment:
How about to use following one maven command instead of two?
mvn clean install javadoc:javadoc

Reply:
or use
mvn clean install && mvn javadoc:javadoc